### DISCUSSION TITLE
Unable to re-import same pcap after `sudo so-nsm-clear`

### DISCUSSION THREAD

**Discussion by silversub** [Security-Onion-Solutions/securityonion#7933] [Unsupported Versions] [answered]
I'm unable to re-import the same pcap after running `sudo so-elastic-clear` and `sudo so-nsm-clear`

```
$ sudo so-import-pcap 2022-01-01-thru-03-server-activity-with-log4j-attempts.pcap
Processing Import: /home/admin/2022-01-01-thru-03-server-activity-with-log4j-attempts.pcap
- verifying file
- assigning unique identifier to import: 7a3a2a7ca44516948b347580a90927ab
- this PCAP has already been imported; skipping
```
I had to manually delete the directory `/nsm/import/7a3a2a7ca44516948b347580a90927ab`

Is that expected behavior?

**Comment by TotieBash** [comment]
It appears it is expected behavior because the developer wrote those error code logic so it must be expected (my assumption).  I can confirm that I had that same error last week with using so-import-pcap.  I fixed it by doing the same procedure you did. I had to delete the reference directory in /nsm/import.

**Comment by dougburks** [comment, marked as answer]
Yes, it's expected behavior. If you're repeatedly importing pcaps, then you may want to consider using VM snapshots as I described at https://github.com/Security-Onion-Solutions/securityonion/discussions/7236.
